### PR TITLE
CORE: PATCH /api/articles/:article_id

### PR DESCRIPTION
**Pull request by Kingsl8yer**
CORE: POST /api/articles/:article_id/comments completed with its test

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -180,3 +180,46 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+  test("status 400: responds with error message when given an invalid inc_votes", () => {
+      return request(app)
+          .patch("/api/articles/1")
+          .send({inc_votes: "invalid"})
+          .expect(400)
+          .then(({body}) => {
+              expect(body.msg).toBe("Bad request");
+          });
+  });
+});
```
Comment (elijaharley) [inline __tests__/index.test.js:314]: I'd also add a test here with the wrong key for inc_votes and an empty body.

**Review by elijaharley — APPROVED** [review]
Really nice work on this endpoint - please see my note about a couple of extra tests.

File: __tests__/index.test.js [hunk 2]
```diff
@@ -124,102 +124,118 @@ describe("GET /api/articles", () => {
...
-          });
+        expect(body.length).toBe(11);
+        expect(body[0].article_id).toBe(1);
+        body.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+        });
```
Comment (NRMSMITH) [inline __tests__/index.test.js]: lovely improvement on this test

File: __tests__/index.test.js (hunk 1 above)
Comment (NRMSMITH) [reply __tests__/index.test.js:314]: Great implementation of the feedback.

**Review by NRMSMITH — APPROVED** [review]
Happy for you to merge 🚀
